### PR TITLE
there is one JSON file per ABAP object

### LINKED ISSUE #276: file name <name>.<type>.json exists for each object
add/amend to documentation

### PR DESCRIPTION
**Pull request by albertmink**
closes #276

### CODE REVIEW

File: docs/json.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## JSON Files in ABAP File Formats
-The ABAP file formats provide JSON schemas for annotation and validation of serialized ABAP objects.
+JSON data files are central, since very ABAP object specifies a JSON file.
+Depending on the object, there might be additional files, but there is always a JSON file.
```
Comment (larshp) [inline docs/json.md:3]:
```suggestion
Depending on the object, there might be additional files, but there is always at least one JSON file.
```

**Comment by larshp** [comment]
suggest repeating the definition from https://github.com/SAP/abap-file-formats/blob/main/docs/specification.md#file-names along with an example

**Comment by schneidermic0** [comment]
I agree, we should specify (at least) the pattern of the filename:

`<object_name>.<object_type>.json`

An example is helpful as well.

File: docs/json.md (hunk 1 above)
Comment (albertmink) [reply docs/json.md]: see efa7296

**Review by schneidermic0 — APPROVED** [review]
LGTM

**Pull request merged**